### PR TITLE
Add release highlights

### LINKED ISSUE #11435: Add release highlights section to the Beats documentation
Other products document release highlights. We should do the same for Beats.

### PR DESCRIPTION
**Pull request by dedemorton**
Adds release highlights based on the 7.0 GA blog, with some slight modifications to language and style to make the content to make it work better in the docs.

I also moved the breaking changes and release notes so they are parallel to the ES and Kibana docs.

Closes #11435

### CODE REVIEW

**Comment by dedemorton** [comment]
@lcawl FYI that I've started the Beats release highlights for 7.0

**Comment by dedemorton** [comment]
Marked as "in progress" because I want to work more details from the Beats GA blog into this topic.

File: libbeat/docs/highlights-7.0.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+In 6.6, {es} added advanced capabilities for index management. Rather than
+simply performing management actions on your indices on a set schedule, you can
+base actions on other factors such as shard size and performance requirements.
+You control how indices are handled as they age by attaching a lifecycle policy
+to the index template used to create them. You can update the policy to modify
+the lifecycle of both new and existing indices. This set of capabilities are
+grouped in the {ref}/index-lifecycle-management.html[index lifecycle management
+(ILM)] APIs.
+
+In 7.0, Beats default to rotate indices by using ILM policies, if the {es}
```
Comment (webmat) [inline libbeat/docs/highlights-7.0.0.asciidoc]: Nit: `In 7.0, {beats} defaults` (note the added trailing s)

File: libbeat/docs/highlights-7.0.0.asciidoc [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+management UI, or directly via the {es} API.
+
+[float]
+==== Stack monitoring
+
+The full suite of modules to {stack-ov}/monitoring-production.html[monitor your
+{stack}] are now GA. These include the {metricbeat} modules for {es}, {ls}, and
+{kib}.
+
+In the future, we will switch to {metricbeat} as the recommended shipper
```
Comment (webmat) [inline libbeat/docs/highlights-7.0.0.asciidoc]: Not sure how closely you want to follow the blog post. But I would write "recommended agent" instead.

File: libbeat/docs/highlights-7.0.0.asciidoc (hunk 1 above)
Comment (webmat) [reply libbeat/docs/highlights-7.0.0.asciidoc]: Actually perhaps even "In 7.0, {beats} defaults to rotating"

Comment (dedemorton) [reply libbeat/docs/highlights-7.0.0.asciidoc]: Yah, I agree. Didn't sound right to my ear either.

File: libbeat/docs/highlights-7.0.0.asciidoc [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-highlights[]
+
+[float]
+==== Elastic Common Schema (ECS)
+
+The {ecs-ref}/index.html[Elastic Common Schema], or ECS, is an open source
+specification that defines a common set of document fields for event data
```
Comment (urso) [inline libbeat/docs/highlights-7.0.0.asciidoc]: nit: open source specification sounds weird to me. Would make open standard sense here?

@webmat WDYT?

File: libbeat/docs/highlights-7.0.0.asciidoc [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+You control how indices are handled as they age by attaching a lifecycle policy
+to the index template used to create them. You can update the policy to modify
+the lifecycle of both new and existing indices. This set of capabilities are
+grouped in the {ref}/index-lifecycle-management.html[index lifecycle management
+(ILM)] APIs.
+
+In 7.0, {beats} defaults to rotating indices by using ILM policies, if the {es}
+version to which they connect supports ILM. The default policy rotates indices
+when they reach 50 GB or 30 days. You can edit the ILM policy by using the {kib}
+management UI, or directly via the {es} API.
```
Comment (urso) [inline libbeat/docs/highlights-7.0.0.asciidoc]: fyi; default policy name is `<beatname>-<version>`

File: libbeat/docs/highlights-7.0.0.asciidoc (hunk 3 above)
Comment (webmat) [reply libbeat/docs/highlights-7.0.0.asciidoc]: @urso It's the wording we've been using in a bunch of places. I agree it sounds a bit weird.

However I would not change it to open standard: although we're very open to external contributions, ECS is not being run like an open standard.

Comment (dedemorton) [reply libbeat/docs/highlights-7.0.0.asciidoc]: I'm leaving as-is to match the blog, but we can always revisit the decision later when we truly have an "open standard"

File: libbeat/docs/highlights-7.0.0.asciidoc (hunk 4 above)
Comment (dedemorton) [reply libbeat/docs/highlights-7.0.0.asciidoc]: Naming seems fairly intuitive so let's not mention it here in case we change the name later on.

**Pull request merged**